src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -175,10 +207,20 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
   }
 
   private case class SerializedAtomicWrite(persistenceId: String, payload: Seq[Serialized])
-  private case class Serialized(sequenceNr: Long, serialized: ByteBuffer)
+  private case class Serialized(sequenceNr: Long, serialized: ByteBuffer, tags: Set[String])
   private case class PartitionInfo(partitionNr: Long, minSequenceNr: Long, maxSequenceNr: Long)
 }
 
+object CassandraJournal {
+
+  private[cassandra] val timeBucketFormatter: DateTimeFormatter = DateTimeFormatter.ofPattern("yyyyMMdd")
+
+  private[cassandra] def timeBucket(epochTimestamp: Long): String = {
```

Comment:
The time bucket should be a type other than String and handled consistencly across the codebase.